spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -489,10 +479,10 @@ public void addPartitionToPartitionedSnapshotIdInheritanceEnabledInTwoRuns() {
     Pattern uuidPattern = Pattern.compile("[a-f0-9]{8}(?:-[a-f0-9]{4}){4}[a-f0-9]{8}");
 
     Matcher matcher = uuidPattern.matcher(manifestPath);
-    Assert.assertTrue("verify manifest path has uuid", matcher.find());
+    Assertions.assertThat(matcher.find()).as("verify manifest path has uuid").isTrue();
```

Comment:
```suggestion
    assertThat(matcher).as("verify manifest path has uuid").matches();`
```

Reply:
`matches` is a full string match, so this couldn't pass the test if this test was changed. Or, change the regex pattern.